CHANGELOG/CHANGELOG-2.1.md

```diff
@@ -0,0 +1,73 @@
+# Changelog 2.1
+
+## 2.1.0 - 2024-07-26
+
+### Added
+
+- Add a new feature `BGPPolicy` to allow users to initialize BGP process on selected Kubernetes Nodes and advertise Service IPs, Pod IPs, and Egress IPs to remote BGP peers.
+  - Add new `BGPPolicy` API and CRD Controller. ([#6009](https://github.com/antrea-io/antrea/pull/6009) [#6203](https://github.com/antrea-io/antrea/pull/6203), [@hongliangl])
+  - Add BGP datapath interface and implement goBGP integration. ([#6447](https://github.com/antrea-io/antrea/pull/6447), [@hongliangl])
+  - Add documentation for the new `BGPPolicy` feature. ([#6524](https://github.com/antrea-io/antrea/pull/6524), [@hongliangl])
+  - Refer to [this document](https://github.com/antrea-io/antrea/blob/release-2.1/docs/bgp-policy.md) for more information about this feature.
+- Add a new feature `NodeLatencyMonitor` to allow users to do east/west connectivity monitoring and provide the latency query API.
+  - Add `NodeLatencyMonitor` feature to do ping mesh in antrea-agent and monitor the latency between Nodes. ([#6120](https://github.com/antrea-io/antrea/pull/6120), [@IRONICBo])
+  - Export Node latency query API definition. ([#6392](https://github.com/antrea-io/antrea/pull/6392), [@IRONICBo])
+- Add two new antctl commands to ensure K8s Cluster meets requirements before Antrea installation, and allow users to validate basic network functionality after Antrea is installed.
+  - Add `antctl check cluster` command to to ensure that the K8s cluster is configured properly before Antrea installation. ([#6278](https://github.com/antrea-io/antrea/pull/6278), [@kanha-gupta])
+  - Add `antctl check installation` command to conduct Pod/Service connectivity checks and verify basic NetworkPolicy rules. ([#6133](https://github.com/antrea-io/antrea/pull/6133) [#6313](https://github.com/antrea-io/antrea/pull/6313) [#6367](https://github.com/antrea-io/antrea/pull/6367), [@kanha-gupta])
+  - Add documentation for the new `antctl check` commands. ([#6373](https://github.com/antrea-io/antrea/pull/6373), [@kanha-gupta])
+  - Refer to [this document](https://github.com/antrea-io/antrea/blob/release-2.1/docs/antctl.md#performing-checks-to-facilitate-installation-process) for more information about these new commands.
+
+### Changed
+
+- Ensure full functionality of AntreaProxy with proxyAll enabled when kube-proxy presents. ([#6308](https://github.com/antrea-io/antrea/pull/6308), [@hongliangl])
```

Comment:
"Ensure Antrea Proxy handles all Service traffic with proxyAll enabled, even when kube-proxy is present"?